libs/test/SalesforceSDKTest/src/com/salesforce/androidsdk/security/KeyStoreWrapperTest.java

```diff
@@ -224,6 +224,7 @@ private void tryNewOrUpgradedClientAgainstNewOrOldServer(boolean newClient, bool
                         ? Encryptor.CipherMode.RSA_OAEP_SHA256 // new server / cipher mode
                         : Encryptor.CipherMode.RSA_PKCS1       // old server / cipher mode
         );
+        
```

Comment:
To trigger the tests ;-)

Reply:
I just made myself a note to change CI to run SalesforceSDK tests if SalesforceAnalytics is modified.